src/wilson/dom.cljs

```diff
@@ -3,21 +3,48 @@
   (:require [wilson.utils :refer [capitalize]]
             [clojure.string :as string]))
 
+(defn describe-key
+  [k]
+  (if (keyword? k)
+      (capitalize k)
+      (::descr (meta k))))
+
+(defn prepare-keys
+  [ks]
+  "Prepares keys for use with wilson.dom/table.
+  `ks` should be a collection of singular keys or vectors of keys
+  pointing at nested data. Vectors are converted into function
+  which (when run on data) will return the data they are poiting at.
```

Comment:
pointing

You should probably say "vectors are interpreted as paths to the relevant data as per `get-in`"